hackathon_site/hardware/views.py

```diff
@@ -59,6 +60,12 @@ class OrderListView(generics.ListAPIView):
         "POST": OrderCreateSerializer,
     }
 
+    filter_backends = (filters.DjangoFilterBackend, OrderingFilter)
+    filterset_class = OrderFilter
+    ordering_fields = "created_at"
+
+    permission_classes = (UserHasHardareOrderViewPermission,)
+
```

Comment:
Since there appears to be a search box on figma, can you add a [search filter](https://www.django-rest-framework.org/api-guide/filtering/#searchfilter) and search fields as well? We're doing that with the other endpoints too, my bad for not updating the ticket when we started doing it.

Some search fields we might want, from looking at the Figma:
- Order ID
- Team code

Then you just need one search that does for example `_build_filter_url(search="<first 4 characters of a team code>"`. To make sure you don't run into any random issues with the randomly generated team codes, you can set the team codes to be AAAA, BBBB, etc in the setup method.